### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by mc-kaishixiaxue**

### CODE REVIEW

File: src/main/resources/assets/tropicraft/lang/zh_CN.lang [hunk 1]
```diff
@@ -369,26 +414,485 @@ entity.tropicraft.koa.trade.not_enough_time=在没有混合气体的情况下，
...
+tropicraft.encyclopedia.section.nature.title=热带植物
+tropicraft.encyclopedia.section.decor.title=装饰
+tropicraft.encyclopedia.section.flowers.title=热带花卉
+tropicraft.encyclopedia.section.shells.title=贝壳
+tropicraft.encyclopedia.section.music.title=热带音乐
+
+tropicraft.encyclopedia.section.misc.title=杂项
+tropicraft.encyclopedia.section.lovetropics.title=热爱热带
+
+tropicraft.encyclopedia.lt17.title=热爱热带 2017
```
Comment (3TUSK) [inline src/main/resources/assets/tropicraft/lang/zh_CN.lang:468]: 我不是很确定是不是应该翻译这个活动的名字……

File: src/main/resources/assets/tropicraft/lang/zh_CN.lang [hunk 2]
```diff
@@ -256,18 +260,50 @@ item.tropicraft.yellow_scuba_tank.name=黄色潜水氧气瓶
...
+tropicraft.cooked=熟的
 
 # Cocktail localizations
 tropicraft.drink.lemonade.name=柠檬汁
 tropicraft.drink.limeade.name=酸橙汁
 tropicraft.drink.orangeade.name=橙汁
 tropicraft.drink.caipirinha.name=鸡尾酒
 tropicraft.drink.coconut_water.name=椰子汁
-tropicraft.drink.pina_colada.name=Piña Colada
+tropicraft.drink.pina_colada.name=椰林飘香
```
Comment (3TUSK) [inline src/main/resources/assets/tropicraft/lang/zh_CN.lang:306]: 这个译名是怎么来的？

File: src/main/resources/assets/tropicraft/lang/zh_CN.lang [hunk 3]
```diff
@@ -369,26 +414,485 @@ entity.tropicraft.koa.trade.not_enough_time=在没有混合气体的情况下，
...
+tropicraft.chat.teleport.unable=在传送之前你应该来一杯椰林飘香！
 
 # Loading Screen
 tropicraft.loading.title.greeting=正在漂流到热带地区...
 tropicraft.loading.title.farewell=祝你活下去...
 
 tropicraft.loading.knowledge.pre=你知道吗？
-tropicraft.loading.knowledge.1=%s，热带地区是Koa部落的聚集地
-tropicraft.loading.knowledge.2=%s，海龟会在傍晚的海滩上下蛋
+tropicraft.loading.knowledge.1=%s，热带地区是原始人部落的聚集地
```
Comment (3TUSK) [inline src/main/resources/assets/tropicraft/lang/zh_CN.lang:430]: Koa 是个部落名吗……？

**Review by 3TUSK — COMMENTED** [review]
I guess it is mostly fine. Some manual entries are untranslated, but I don't think that hurts much. So... a couple of questions, out of curiosity:

File: src/main/resources/assets/tropicraft/lang/zh_CN.lang (hunk 2 above)
Comment (mc-kaishixiaxue) [reply src/main/resources/assets/tropicraft/lang/zh_CN.lang:306]: https://baike.baidu.com/item/%E6%A4%B0%E6%9E%97%E9%A3%98%E9%A6%99

File: src/main/resources/assets/tropicraft/lang/zh_CN.lang (hunk 3 above)
Comment (mc-kaishixiaxue) [reply src/main/resources/assets/tropicraft/lang/zh_CN.lang:430]: According to `entity.tropicraft.koa.name=原始人` and `entity.tropicraft.koa.name=Koa`

File: src/main/resources/assets/tropicraft/lang/zh_CN.lang (hunk 1 above)
Comment (mc-kaishixiaxue) [reply src/main/resources/assets/tropicraft/lang/zh_CN.lang:468]: It doesn't matter.Most players can't care the details.

**Comment by mc-kaishixiaxue** [comment]
I think it's satisfied for Chinese players to play Tropicraft easier.

File: src/main/resources/assets/tropicraft/lang/zh_CN.lang [hunk 4]
```diff
@@ -256,18 +260,50 @@ item.tropicraft.yellow_scuba_tank.name=黄色潜水氧气瓶
...
+item.tropicraft.tropicalfish.0.cooked.name=熟小丑鱼
+item.tropicraft.tropicalfish.1.cooked.name=熟女王天使鱼
+item.tropicraft.tropicalfish.2.cooked.name=熟黄高鳍刺尾鱼
+item.tropicraft.tropicalfish.3.cooked.name=熟蝴蝶鱼
+item.tropicraft.tropicalfish.4.cooked.name=熟红珍珠关刀
+item.tropicraft.tropicalfish.5.cooked.name=熟斗鱼
+item.tropicraft.tropicalfish.6.cooked.name=熟帝王刺尾鲷
+item.tropicraft.tropicalfish.7.cooked.name=熟皇家丝鲈
+
+item.tropicraft.riverfish.piranha.raw.name=生水虎鱼
```
Comment (liach) [inline src/main/resources/assets/tropicraft/lang/zh_CN.lang:291]: `食人鱼`?

Comment (mc-kaishixiaxue) [reply src/main/resources/assets/tropicraft/lang/zh_CN.lang:291]: https://baike.baidu.com/item/水虎鱼/50738

**Pull request merged**